src/index.ts

```diff
@@ -190,7 +190,7 @@ const getJsImports = (code: string, absolutePath: string, pathResolver: (string)
 
     if (importStart >= 0) {
         const imports = parseImports(code.substr(code.indexOf('import')))
-            .filter(i => !i.fromModule.startsWith('\\u0000'))
+            .filter(i => !checkForBom(i.fromModule))
```

Comment:
Using a more declarative name for this function would make this line more readable. Something like `!hasBom(i.fromModule)`.

Reply:
got it! will fix